actionpack/actionpack.gemspec

```diff
@@ -34,7 +34,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency "activesupport", version
 
-  s.add_dependency "rack",      "~> 2.0"
+  s.add_dependency "rack",      "~> 2.0", ">= 2.1.0"
```

Comment:
I merged that change on 2-0-stable as well so we don't need to force 2.1. I'll release that version so we can remove this.